lesson 0/README.md

```diff
@@ -1,18 +1,27 @@
 # Lesson 0 - Getting prepared
 
-Welcome to [insert class name here]! We're excited to be with you this semester, and in order to have a smoother process there are a few things that we'd like you to do before coming in to class on September 1st.
+Welcome to MobileProto! We're excited to be with you this semester, and in order to have a smoother process there are a few things that we'd like you to do before coming in to class on September 1st.
 
-## 1. Download Android Studio
+## 1. Download and Install Android Studio
 
-If you do this before class we'll have a lot more time to help you out and get started! Please download Android Studio 2.x [link here].
+If you do this before class we'll have a lot more time to help you out and get started! Please download [Android Studio 2.1](https://developer.android.com/studio/install.html).
 
 *Note*: For those of you who already have Android Studio, it's important that you have the latest version. This makes it easier for us to help you with any problems you might have.
-*Note 2*: If you choose to use Android Studio on Windows, then you'll need to download github desktop [link here].
 
-## 2. Set up your github account with Android Studio
+## 2. Set up github
 
-Click some buttons, enter your account name, and voila! Git integration! Super useful. If you don't have a git account... make one!
+Put your github acocunt name on [this]() spreadsheet. [Fork](https://help.github.com/articles/fork-a-repo/) our repo to your account. Finally, clone the repository and add our's as a remote named `upstream`:
 
-## 3. Join our slack channel
+```
+cd some_directory  # wherever you want to store your MobileProto repo
+git clone git@github.com:YOUR_ACCOUNT_NAME/Mobile-Proto-16.git  # this should be the ssh clone address given to you by github
+git remote add upstream git@github.com:MobileProto16/Mobile-Proto-16.git
+```
+
+As an aside, Android studio has git integration. Be on the lookout for this. It's pretty cool.
```

Comment:
In the spirit of keeping our docs as concise as possible, should we remove this?